src/test/ui/parser/recover-where-clause-before-tuple-struct-body.rs

```diff
@@ -0,0 +1,18 @@
+// Regression test for issues #100790 and #106439.
```

Comment:
Could this test be marked `// run-rustfix`? I'm not sure the missing `;` has a tool only suggestion, but otherwise this test seems like it could be checked for the suggestions keeping to apply cleanly.

Reply:
Should I make the diagnostic *machine-applicable*, too? It currently is *maybe-incorrect*.

Reply:
Yeah, do so too.